services/QuillLMS/spec/models/activity_spec.rb

```diff
@@ -603,4 +603,26 @@
       end
     end
   end
+
+  context '#activity_with_recommendations_ids' do
+    let!(:recommendation1) { create(:recommendation) }
+    let!(:recommendation2) { create(:recommendation) }
+
+    after do
+      Rails.cache.clear
```

Comment:
I'm not sure I understand the purpose of this after hook. Is there some cache state we're trying to avoid in line 616?

Reply:
That makes sense.  Thank you for the detailed explanation.